### PR TITLE
Bump to v2.3.0

### PR DESCRIPTION
**Pull request by maltoe**

### CODE REVIEW

**Review by agatheblues — APPROVED** [review]
Looks good, thanks 🌵 !

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,34 +1,24 @@
 # Changelog rubocop-bitcrowd
 
-Presented in reverse chronological order.
+Presented in reverse chronological order. Document adheres to [keepachangelog.com](https://keepachangelog.com/en/1.0.0/), though our sections are called "Deprecations", "Potentially breaking changes", "New features", and "Fixes".
```
Comment (klappradla) [inline CHANGELOG.md:3]: 💚

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,34 +1,24 @@
...
 
-## main
+## Unreleased
 
-https://github.com/bitcrowd/rubocop-bitcrowd/compare/v2.2.0...HEAD
+https://github.com/bitcrowd/rubocop-bitcrowd/compare/v2.3.0..HEAD
```
Comment (klappradla) [inline CHANGELOG.md]: Is it on purpose that you switched to `..` from `...`? The two dots means one will only see the file changes, not the commits.

**Review by klappradla — APPROVED** [review]
Thanks for the cleanup 💚

File: CHANGELOG.md (hunk 2 above)
Comment (maltoe) [reply CHANGELOG.md]: Oh really? Didn't know that. Thought the behaviour is akin to `git log`. I'll merely aligned these lines, but will change them back to `...` now.

**Comment by maltoe** [comment]
@klappradla can you run the `release` rake task and push the gem please 🙏 ? I don't have access to bitcrowd's rubygems account and can't find it in 1pw.

**Pull request merged**